src/app/course/event/course-event-create/course-event-create-edit.component.ts

```diff
@@ -83,6 +84,7 @@ export class CourseEventCreateEditComponent implements OnInit {
      * @param formData - grabs the components formData and creates a request based on that
      */
     async submitEvent(formData: FormGroup) {
+        this.clicked = true
```

Comment:
There is a problem with this logic. If for any reason the user doesn't get redirected after clicking they can't click again. For example, they might have put an invalid value for something and they receive an error from the back end. Then when they fix it they can't click again. We have to make sure we change this back to false when the request is over.
You can do it in the `.subcribe` in `onError`.